src/sniffer.c

```diff
@@ -1067,10 +1065,21 @@ static int LoadKeyFile(byte** keyBuf, word32* keyBufSz,
     XFCLOSE(file);
 
     if (typeKey == SSL_FILETYPE_PEM) {
-        saveBuf = (byte*)malloc(fileSz);
+        byte* saveBuf   = (byte*)malloc(fileSz);
```

Comment:
I know it was already this way, but perhaps the sniffer should use our XMALLOC / XFREE macros so the sniffer is more portable? It looks like the file most uses the X* macros elsewhere.